src/utils/clmfates_interfaceMod.F90

```diff
@@ -1608,7 +1610,8 @@ subroutine wrap_update_hlmfates_dyn(this, nc, bounds_clump,      &
           z0m(col%patchi(c)+1:col%patchf(c)) = 0.0_r8
           displa(col%patchi(c)+1:col%patchf(c)) = 0.0_r8
           dleaf_patch(col%patchi(c)+1:col%patchf(c)) = 0.0_r8
-
+          wesley_pft_index_patch(col%patchi(c)+1:col%patchf(c)) = 0
```

Comment:
Can you do `0.0_r8` here, like it is done everywhere?

Reply:
In DryDepVelocity the initital valaue is `-1` and there are checks `<0`.

Reply:
These are defined at integers in Canopystatemod so I think they should just be '0'? 

Would it be good form to start this as the unset values of -1? Or just stick to 'zaeroing'?